ThinBridge/Chronos
# Remove the print as pdf function

# Which issue(s) this PR fixes:

https://github.com/ThinBridge/Chronos-SG/issues/143

# What this PR does / why we need it:

"Print as pdf" doesn't work correctly for PDF files. Also, it duplicates to "Print" -> "Output as PDF".
We should remove it from the menu bar and context menu.

Supplementary information: there is no shortcut key like `Ctrl + P` to execute "Output as PDF".

![menubar](https://github.com/user-attachments/assets/985e7844-cfb6-4a8d-8bcf-819731492c49)

![contextmenu](https://github.com/user-attachments/assets/9c0821e8-4cb5-4804-a314-05a90d644bfc)


# How to verify the fixed issue:

## The steps to verify:

* Click "File" on the menu bar
  * [x] Confirm that "Print as pdf" is not listed in the menu
* Open some site like https://www.google.co.jp/.
* Right click on a blank space
  * [x] Confirm that "Print as pdf" is not listed in the context menu
* Right click on a link
  * [x] Confirm that "Print as pdfF" is not listed in the context menu
* Right click on a text box (input)
  * [x] Confirm that "Print as pdf" is not listed in the context menu
* Right click on an image
  * [x] Confirm that "Print as pdf" is not listed in the context menu
